ios/FluentUI/Tab Bar/TabBarItem.swift

```diff
@@ -7,6 +7,13 @@ import UIKit
 
 @objc(MSFTabBarItem)
 open class TabBarItem: NSObject {
+    @objc(MSFTabBarItemBadgeStyle)
+    public enum BadgeStyle: Int {
```

Comment:
I think `pod lib lint` is failing because of the `BadgeStyle` getting moved to the `TabBarItem`. Maybe we should just add a `var` in `TabBarItem`, and keep the style in the `BadgeLabel`?